addons/paint-by-default/addon.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Paint costume by default",
-  "description": "Changes the default action of \"Choose a Sprite/Costume/Backdrop/Sound\" buttons, which open the library by default.",
+  "name": "\"Paint costume\" by default",
```

Comment:
"Paint costume" is not the name of any Scratch feature, so it doesn't need quotes.